<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>622</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>808</width>
    <height>622</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 226, 183);</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bonello,regular</family>
           <pointsize>25</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">  font-family: 'Bonello', regular;
  font-weight: bold;
  letter-spacing: .2rem;</string>
         </property>
         <property name="text">
          <string>Cadastrar Usuário</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="btnCad_User">
         <property name="styleSheet">
          <string notr="true">  background:white;
letter-spacing: .2rem;
  color: black; /* Texto branco */
  font-family: 'Betty', regular;
  font-size: 16px;
  font-weight: bold;
  padding: 12px 24px; /* Tamanho do botão */
  border: none; /* Remove bordas padrão */

  border-radius: 8px; /* Bordas arredondadas */
  cursor: pointer; /* Cursor de ponteiro ao passar o mouse */
  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1); /* Sombra suave */
  transition: all 0.3s ease-in-out; /* Animação suave */</string>
         </property>
         <property name="text">
          <string>Entrar</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Bonello,regular</family>
           <pointsize>25</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">  font-family: 'Bonello', regular;
  font-weight: bold;
  letter-spacing: .2rem;</string>
         </property>
         <property name="text">
          <string>Cadastrar Livro</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btnCad_Livro">
         <property name="styleSheet">
          <string notr="true">  background:white;
letter-spacing: .2rem;
  color: black; /* Texto branco */
  font-family: 'Betty', regular;
  font-size: 16px;
  font-weight: bold;
  padding: 12px 24px; /* Tamanho do botão */
  border: none; /* Remove bordas padrão */

  border-radius: 8px; /* Bordas arredondadas */
  cursor: pointer; /* Cursor de ponteiro ao passar o mouse */
  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1); /* Sombra suave */
  transition: all 0.3s ease-in-out; /* Animação suave */</string>
         </property>
         <property name="text">
          <string>Entrar</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Bonello,regular</family>
           <pointsize>25</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">  font-family: 'Bonello', regular;
  font-weight: bold;
  letter-spacing: .2rem;</string>
         </property>
         <property name="text">
          <string>Empréstimo Livro</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btnEmprestimo">
         <property name="styleSheet">
          <string notr="true">  background:white;
letter-spacing: .2rem;
  color: black; /* Texto branco */
  font-family: 'Betty', regular;
  font-size: 16px;
  font-weight: bold;
  padding: 12px 24px; /* Tamanho do botão */
  border: none; /* Remove bordas padrão */

  border-radius: 8px; /* Bordas arredondadas */
  cursor: pointer; /* Cursor de ponteiro ao passar o mouse */
  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1); /* Sombra suave */
  transition: all 0.3s ease-in-out; /* Animação suave */</string>
         </property>
         <property name="text">
          <string>Entrar</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
